utility/src/main/java/org/oppia/util/firebase/CrashLoggerImplementation.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.util.firebase
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import javax.inject.Singleton
+
+/** Logger for providing custom crash reporting to Firebase Crashlytics. */
+@Singleton
```

Comment:
Since this class isn't injected, this scope won't do anything. It can be removed.